kartothek/io_components/gc.py

```diff
@@ -10,7 +10,7 @@ def dispatch_files_to_gc(dataset_uuid, store_factory, chunk_size, factory):
         dataset_uuid=dataset_uuid,
         store=store_factory,
         factory=factory,
-        load_dataset_metadata=False,
```

Comment:
@jochen-ott-by As this is gc, I think we should set it to `False` here.

Reply:
I think it does not really matter and we can use `load_dataset_meadata=True` everywhere.